### PR TITLE
Add support for MX records

### PR DESCRIPTION
**Pull request by Krivoblotsky**

### CODE REVIEW

File: Tests/DNSTests/DNSTests.swift [hunk 1]
```diff
@@ -80,6 +81,27 @@ class DNSTests: XCTestCase {
 
         XCTAssertEqual(nameServer, nameServer0Copy)
     }
+    
+    func testMXRecord() {
```
Comment (Bouke) [inline Tests/DNSTests/DNSTests.swift:85]: Let's keep naming consistent with other cases: `testMailExchangeRecord`

**Review by Bouke — CHANGES_REQUESTED** [review]
Nice! Could you change this method name?

**Comment by Krivoblotsky** [comment]
Sure, done.
https://github.com/Bouke/DNS/pull/32/commits/f33405dec06b3a6920d925998a573e6e0da8bec5

**Comment by Bouke** [comment]
Thanks!

**Pull request merged**